backend/data/src/main/kotlin/io/tolgee/model/Project.kt

```diff
@@ -125,6 +125,9 @@
   @ColumnDefault("true")
   override var icuPlaceholders: Boolean = true
 
+  @ColumnDefault("false")
+  var useNamespaces: Boolean = false
```

Comment:
You have to add @ActivityLoggedProp and also check the activity log in the UI whether all is rendered correctly.

Reply:
Yes, thank you, I debugged to it the hard way :smile: